mmv1/third_party/terraform/services/composer/resource_composer_environment.go.erb

```diff
@@ -583,9 +583,15 @@ func ResourceComposerEnvironment() *schema.Resource {
 								Schema: map[string]*schema.Schema{
 									"machine_type": {
 										Type:         schema.TypeString,
-										Required:     true,
+										Optional:     true,
```

Comment:
I think this change should be reflected in mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown